README.md

```diff
@@ -1,3 +1,252 @@
-# `flytelab`
+<html>
+    <p align="center"> 
+        <img src="https://github.com/flyteorg/flyte/blob/master/rsts/images/flyte_circle_gradient_1_4x4.png" alt="Flyte Logo" width="100">
+    </p>
+    <h1 align="center">
+        Flytelab
+    </h1>
+    <p align="center">
+        The Open Source Repository of Flyte-based Projects
+    </p>
+    <p align="center">
+        <a href="https://slack.flyte.org">
+            <img src="https://img.shields.io/badge/slack-join_chat.svg?logo=slack&style=social" alt="Slack" />
+        </a>
+    </p>
+</html>
 
-The open source repository of machine learning projects using [Flyte](https://flyte.org/).
+The purpose of this repository is to showcase [Flyte's](https://flyte.org/) capabilities in end-to-end
+applications that do some form of data processing or machine learning.
+
+The source code for each project can be found in the `projects` directory, where each project has its
+own set of dependencies.
+
+## 🚀 Create a New Project
+
+> **NOTE:** Make sure you're using `Python > 3.7`
+
+We use `cookiecutter` to manage project templates.
+
+> **NOTE:** For [MLOps Community Engineering Labs Hackathon](https://flyte.org/hackathon/) participants:
+> Each team will have its own branch on the main `flyteorg/flytelab` repo. If you're part of a team of more
+> than one person, assign one teammate to create a project directory and push it into your team's branch.
+
+Install prerequisites:
```

Comment:
Shall we talk about creating a branch and virtualenv? The note talks about creating a branch, but I believe it'd be clear to include that as part of the prerequisites.

Moreover, if the user isn't a committer, should we mention that "fork" is how the user has to start working on a Flytelab project?

Reply:
> Shall we talk about creating a branch and virtualenv

the environment that creates the project is different from the environment that runs the project workflows, so I didn't want to bog down/confuse newcomers with having to create a virtual environment just to install `cookiecutter` on it, then creating _another_ virtual environment for the project. I think this is a nice-to-have but not really necessary.

> The note talks about creating a branch, but I believe it'd be clear to include that as part of the prerequisites

👍🏽

> Moreover, if the user isn't a committer, should we mention that "fork" is how the user has to get started?

cool, will add